index.html

```diff
@@ -24,7 +24,7 @@
             <div class="introc-content pad-aside">
```

Comment:
Good job on your work. 
But I think it would be better to use semantic tags properly because they make your code better.